apps/omg_child_chain/lib/omg_child_chain/block_queue/core.ex

```diff
@@ -599,12 +599,12 @@ defmodule OMG.ChildChain.BlockQueue.Core do
   end
 
   defp log_known_tx(submission) do
-    _ = Logger.debug("Submission #{inspect(submission)} is known transaction - ignored")
+    _ = Logger.warn("Submission #{inspect(submission)} is known transaction - ignored")
```

Comment:
didn't this warn us of a certain condition (the one where private keys got mismatched between environments?)

Reply:
oh my, good point :thinking:. However, IIRC, that log was just something we investigated as strange, but I suspect you'll see it on any occasion where a submission isn't mined and gets re-submitted. We just never saw it because `:debug`.

So, yes, it did provide some info, but I'd argue this was more of a coincidence.

In the end, not sure what to do about this. We can try and see, and if it is too chatty we revert to `debug` and possibly think of other warnings/checks that could be more accurate. Let's just make a self-note to not forget to check the chatiness.

Reply:
put back to :debug as it was spamming tests :)